Automattic/simplenote-ios
# Contextual Menu Support

### Details:
In this PR we're implementing the contextual UITableView / UIMenu API, which replaced the legacy 3d Touch API(s).

@etoledom Sir!! would you be game for a quick PR?
Thanks in advance!!

Ref. #863

### Test: All Notes
1. Log into your account
2. Select All Notes
3. Long press over any Note in your List

- [x] Verify that a Preview with actions at the bottom shows up
- [x] Verify that pressing the Preview pushes the Editor
- [x] Verify that **Copy Link** places in the Pasteboard a link to the selected note (paste elsewhere!)
- [x] Verify that pressing **Share** opens the Share Sheet
- [x] Verify that pressing **Delete** trashes the selected note

### Test: Trash
1. Log into your account
2. Open the Trash
3. Long press over any trash ed notes

- [x] Verify nothing happens!


### Release
These changes do not require release notes.

### Screenshot

<img width="300" src="https://user-images.githubusercontent.com/1195260/95348350-5473f680-0894-11eb-9c52-c0bf22a83b8e.png">
